### ISSUE TITLE
Information about Trustroots Foundation is not accurate any more

### ISSUE THREAD

**Issue by mariha** [Trustroots/trustroots#2585]
As of March 2022 [Trustroots Foundation has been dissolved](https://find-and-update.company-information.service.gov.uk/company/09489825/filing-history). While we still try to find out what exactly it means for the community, we need to update the website with more accurate information.

Proposed changes:
- [x] Add a note at the top of the [Foundation](https://www.trustroots.org/foundation) page:
   > As of May 2022 [Trustroots Foundation has been dissolved](https://find-and-update.company-information.service.gov.uk/company/09489825/filing-history). We still try to find out what exactly it means for the community.
- [x] Add a paragraph at the beginning of [FAQ: Foundation - What's your legal status?](https://www.trustroots.org/faq/foundation#what-is-your-legal-status) question:
   > As of May 2022 [Trustroots Foundation has been dissolved](https://find-and-update.company-information.service.gov.uk/company/09489825/filing-history). What does it mean for the community? We don't know exactly yet!
- [x] Remove [FAQ: Foundation - Who are the board?](https://www.trustroots.org/faq/foundation#who-are-the-board) question and answer.
- [ ] maybe: Update [front page](https://www.trustroots.org/), at the moment it states:
  > Owned and operated by Trustroots Foundation, a non-profit registered in the United Kingdom since March 2015.
- [ ] maybe: Review and rename _Foundation_ to _Organization_
  - [ ] [Foundation FAQ](https://www.trustroots.org/faq/foundation), links and path `/faq/foundation` -> `/faq/organization`
  - [ ] Selected links to the [Foundation](https://www.trustroots.org/foundation) page
  - [ ] Mentions of _the foundation_ in the texts

**Comment by chmac** [comment]
What's the rationale behind updating the website?

**Comment by mariha** [comment]
Transparency, for the sake of its own and as a foundation of trust.

[Trustroots Manifesto](https://www.trustroots.org/) states:
> We believe in beauty, simplicity and **transparency**.
(emphasizes is mine)

Personally, when I came to TR I highly appreciated and valued it very much that all that information that often remains private in corporate world (give a look to [TR Privacy Policy](https://www.trustroots.org/privacy), [statistics](https://www.trustroots.org/statistics), being open source), is being openly shared with TR members and everyone else. It builds trust to the organization, that it's honest and true to its values.

Why you wouldn't share that information?

**Comment by chmac** [comment]
I hear your rationale is transparency.

How does the proposed change increase transparency?

**Comment by mariha** [comment]
It's just _being_ transparent. I don't think about _transparency_ as something that can be increased or decreased. The organization either _is_ or _is not_ transparent about something.

Current state of Trustroots organization is that the Foundation has been dissolved. Putting this information next to (outdated) information about the Foundation is being transparent about the changes. Not putting that information is misleading for people who wants to get to know something more about the organization behind Trustroots. The current information that _Trustroots is owned and operated by the Foundation_ does not reflect the reality.

**Comment by chmac** [comment]
> The current information that Trustroots is owned and operated by the Foundation does not reflect the reality.

What is the reality then?

All information about the foundation is already available directly from the source, Companies House. It's all public. How do you decide which information needs to be on the trustroots web site in order to maintain transparency?

**Comment by mariha** [comment]
> What is the reality then?

The reality is that the Foundation is dissolved. As for me, a dissolved foundation can not own or operate anything.

I would like to say that Trustroots website/platform is now owned and operated by the community (of volunteers who tend to be a subset of a community of users), but as to me we are not there yet. 

Trustroots as a [commons-based peer production](https://en.wikipedia.org/wiki/Commons-based_peer_production) is my wishful thinking of what I would like the reality to be.

**Comment by chagai95** [comment]
I think it's nice to be transparent about this as well and I don't think it's a big deal to have it on the website.

**Comment by chmac** [comment]
> The reality is that the Foundation is dissolved. As for me, a dissolved foundation can not own or operate anything.

How do you reach this understanding?

Do you have any resources or links to share which support this view?

Would your perspective on this topic change if this were untrue?

**Comment by mrkvon** [comment]
Callum @chmac,

@mariha has done a lot of work answering your questions. It would be helpful, at this point (or earlier), if you express your concerns instead.

Mariha proposed changes which she thinks describe reality well. It also seems to me that with the limited knowledge we all have, this change reflects actual state of things pretty accurately.

It would be helpful, if you could point out the issues you have. The questions like "what do you think is reality", "how do you reach this understanding", asking for resources to support the obvious, or _what if_ questions like "Would your perspective on this topic change if this were untrue?" imo are not very helpful. They're vague at best, and difficult to answer, and they may (unintentionally) obstruct resolving this issue. Mariha already clearly described what she thinks reality is, both here and in #2588, and supported her view with arguments.

**The burden of pointing out the issues is on your side now**, @chmac . So, **could you point out your concerns or issues with the proposed changes**?

It could be that you think something (specific) is not accurate, or that it's accurate, but publishing would be harmful in some way. (Or something else of course...)

If, for some reason, your arguments shouldn't go public, you can share them in a private conversation (e.g. with Mariha), or join our regular weekly call and share your concerns there. ❤️

**Comment by chmac** [comment]
@mrkvon Thanks for the input, I'm happy to share more about my thinking.

Firstly, my go to policy is do nothing. Cause no work to others, justify all efforts well. Or maybe, disciplined laziness. Whatever. So to me, as I was asked to review the PR, in my mind, the PR needs to justify its existence. Like why should people spend time reviewing this? Why should somebody deploy it? What's the point of this exercise?

With regards to the current state of the foundation specifically, I hear a lot of speculation. My impression is that people with no formal training or qualifications, and little to no experience in these topics, speculate about what the status is. That speculation seems to produce a "oh, something is wrong here" outcome. I don't share that view, and I haven't seen any evidence to support it. In other words, I think it's a mistake. I think people are saying "oh, the foundation was dissolved, something is wrong" when it seems to me that's not true at all. I don't think anything is wrong, I don't think any action is required, and I don't think anything has actually changed in any meaningful way. Of course, I could be wrong about that. Maybe the change does have impact. I'm trying to remain open to this possibility by asking questions about how other people see it.

I also get a sense of "fear-mongering" around the topic. Writing "What does it mean for the community? We don’t know exactly yet!" seems to me to spread fear. When I imagine, how will a prospective user interpret this, I don't imagine it's positive. I imagine people read it and think, "hmm, this weird, something is off here". I don't see any upside to this, and I see potential downside.

From my personal perspective, I wish I'd never mentioned to anybody this topic with the foundation. It seems that by talking about it, I created a shit storm which has only negative consequences. A whole lot of time and energy went into discussing something which, it turns out, is actually a non event.

To summarise, from my perspective this is like a PR to announce that nothing happened, but we're gonna talk about it, and probably cause a bunch of confusion, potentially spark people asking questions, and generally just complicate the issue.

My approach was to try and understand why this is happening, what the thinking behind it is, and see if I can understand where other people's opinions differ from mine, and maybe find some common ground, shared understanding, and clearer picture. But @mrkvon if you think simply stating my own opinion is a more helpful method of communication here, I'm certainly happy to do that.

**Comment by chagai95** [comment]
Hey all, let me try expressing how I see it, hopefully it will be somewhat helpful:

I don't understand much legalese, but from what I understand we had some kind of Trustroots Foundation, this was done to have another hurdle which should help in preventing Trustroots going the wrong way or/and protecting private people from lawsuits. I don't know exactly how this was supposed to help, but apparently it was. Now that there is no foundation, there is a fear that not having a foundation means that this hurdle is potentially not there anymore, making it potentially easier for Trustroots to go the wrong way or making it more risky for the people running the servers of Trustroots.

I think some of us are very sensitive when it comes to hiding information from the community.

Maybe I can suggest we stop the conversation here and @chmac you could spend your time on reviewing other PRs? I understand it's not something you want to spend time on and frankly neither do I, so maybe we can both move to doing other stuff and let people who think this is important do this?


I think if we spend some more time on this, we could adapt the description of the situation so that it will not leave people confused, but if no one wants to make a nicer description then this is what we got, and I think the argument of transparency is a valid one.

**Comment by chmac** [comment]
> "Now that there is no foundation"

I don't think this is accurate. I think you've fundamentally misunderstood what the current situation is.

**Comment by chagai95** [comment]
I updated my comment, sorry for misunderstanding...

**Comment by chagai95** [comment]
Maybe this sentence could be useful to add for example:
```
We are still protected against sellouts by way of the foundation being frozen, possibly a better protection than before. We no longer have an official entity to interface with the paper-world, but that’s a feature we never used and we aren’t planning on using it anytime soon. All good.
```

**Comment by mrkvon** [comment]
@chmac Thank you for expressing your view. I find it very helpful to orient myself in this conversation, and in your point of view.

I also get a better understanding why you ask the questions. I appreciate that you expressed that and listen and try to understand the other side. So perhaps the combination of both asking the questions and sharing the opinion as a starting point could be the most helpful (at least for me)... ❤️ 

Now to some of your concerns:

> Firstly, my go to policy is do nothing. Cause no work to others, justify all efforts well. Or maybe, disciplined laziness. Whatever. So to me, as I was asked to review the PR, in my mind, the PR needs to justify its existence. Like why should people spend time reviewing this? Why should somebody deploy it? What's the point of this exercise?

That makes sense. I'm not sure how it aligns with do-ocracy that Trustroots was designed with. (i.e. If people think a change is needed, they're encouraged to go forward and do it. - at least that's my understanding)
Would it add work to you personally, Callum? (beyond the request to review the PR, which imo is also ok to ignore)

> With regards to the current state of the foundation specifically, I hear a lot of speculation. My impression is that people with no formal training or qualifications, and little to no experience in these topics, speculate about what the status is. [...]

Do you think it would help if we focused on updating just the objective status of Trustroots foundation (i.e. that it is dissolved)? Or do you think it would be harmful to do any change in that direction?

> I also get a sense of "fear-mongering" around the topic.

That's a very valid point. The text "What does it mean for the community? We don’t know exactly yet!" seems to reflect the feeling of volunteers about the topic. There were also plans to announce the dissolution via a newsletter. Anyways, i haven't thought about the effect the particular text could have on potential new members. That's a topic for a further discussion. For now, perhaps the text could be removed or modified to something more reassuring?

---

> I think you've fundamentally misunderstood what the current situation is.

This is interesting. Could you elaborate, Callum? 🙂 We may be tapping the point of misunderstanding here...

**Comment by chmac** [comment]
> > I think you've fundamentally misunderstood what the current situation is.
> 
> This is interesting. Could you elaborate, Callum? 🙂 We may be tapping the point of misunderstanding here...

I don't think anything has changed. I think this was a non event. I don't think it makes any difference at all. It might have some consequence in 6 years time. But for now, it's my understanding that absolutely nothing has changed.

**Comment by chagai95** [comment]
Did you read this message on slack? 🙈
https://trustroots.slack.com/archives/C08SENA9Z/p1659089483340379?thread_ts=1659089483.340379&cid=C08SENA9Z

**Comment by chmac** [comment]
@chagai95 My impression from that post is also "non event".

**Comment by robokow** [comment]
The amusing thing is that TR foundation being dissolved doesn't mean anything to the community or for the community. TR has not changed; the foundation can be brought back into existence. We know the procedure on how to do that. And most important, we know what it entails and means, already. Hence, the wording is incorrect even. Like this, it's just disinformation.

In other words, merging this commit to Master without consensus about it with any admin, simply means it is unlikely to ever get pushed. Just a honest suggestion: @mariha it's wiser to leave the discussion for what it is (for now) and just undo your merge at https://github.com/Trustroots/trustroots/pull/2588.

**Comment by mrkvon** [comment]
@robokow
> The amusing thing is that TR foundation being dissolved doesn't mean anything to the community or for the community. TR has not changed; the foundation can be brought back into existence.

Feel free to propose a patch along these lines. As far as i'm aware, the current status is `Dissolved`, and therefore at least that information is valid and reflects reality well. When the foundation is brought to existence again, the information can be updated, to reflect reality again. Until then, the factual information is valid and correct. The opinionated info (what it means to the community?) is open to a patch, as @mariha already said.

> TR has not changed; the foundation can be brought back into existence. We know the procedure on how to do that. And most important, we know what it entails and means, already.

It would be wonderful if you add that info as part of such patch. AFAIK volunteers, who have extensively discussed this on Slack, have been significantly more uncertain.

> Hence, the wording is incorrect even. Like this, it's just disinformation.

Which wording in particular? Be very welcome to propose a patch.

> In other words, merging this commit to Master without consensus about it with any admin, simply means it is unlikely to ever get pushed.

Can you point out where it's stated that approved requests need to wait for consensus from admins? This PR went through a review process and extensive discussion, and i approved it. Unlike some recent PRs, or direct pushes to master, in which admin authors didn't even bother asking for feedback.

> Just a honest suggestion: @mariha it's wiser to leave the discussion for what it is (for now) and just undo your merge at https://github.com/Trustroots/trustroots/pull/2588.

No, just don't. I think we have a fundamental misunderstanding of a current role of admins in Trustroots, and what's ok and not ok to do with their power.
Here i want to add: People can do mistakes. But they need to own them and say, _It was a mistake, next time we'll do better_

> TR has not changed

It is changing in front of my eyes, unfortunately. I'll be happy to hear your thoughts and share mine in a personal conversation, to which i invited you @robokow

**Comment by robokow** [comment]
@mrkvon admins are here to safeguard TR from other volunteers taking decisions that can bring TR into jeopardy. I think pushing this commit to production does exactly that.

Also, from observing your most recent comments here, I am afraid your contributions are missing a focus. You might want to work on that.

**Comment by mrkvon** [comment]
@robokow 👌🏾 Let's talk about that in person. I'm looking forward to hearing your feedback in more detail, and i'll also share mine.

**Comment by mariha** [comment]
Hello @robokow, nice to meet you!

I am mariha, one of Trustroots volunteer for about two years now, you can see some of my (measurable) contributions here: , nothing special.

As you can also see, @mrkvon happens to be a person who contributed most code to TR (*) from currently active people here.

(*) based on GitHub stats which include only code contributions, obviously there are other contributions too, which are harder to measure and grasp what does not make them less significant and valuable for the community. Being aware of their limits, GH stats acknowledge certain level of commitment and time investment.

**Comment by MrSalami** [comment]
I'd like to add something to this convo onto why it is relevant to our members regarding trust and transparency.

One of the reasons Trustroots was created was as an alternative network to Couchsurfing which broke the trust of hundreds of thousands if not millions of users who use hospitality exchange. Many of those early users left couchsurfing bitterly and came to BeWelcome and to Trustroots.

Those same users are still members on our platform and still remember the trust broken.

So, as an act of transparency, and in good faith we share big news such as we don't have any organization right now - will assure those users when they do find out somehow - whether through media or through word of mouth. 

I find that expressing it strengthens the loyalty to our members.

**Comment by mariha** [comment]
Hello @robokow, nice to meet you!

I am mariha, one of Trustroots volunteers for about two years now, you can see some of my contributions here: https://github.com/Trustroots/trustroots/graphs/contributors.
I also happen to be a former WarmShowers Android app contributor, a person who [asked the app users and other community members to move to TR](https://warmShowers.bike) after WS board went rogue. 

As you can see on that GH stats page, @mrkvon happens to be a person who contributed most code to TR (*) from currently active people here. Others with significant contributions are @simison, @nicksellen, @hlibbabii and others.

I'll be happy to have a chat with you too. 

On [our team's wiki](https://team.trustroots.org/Volunteering.html) you'll find some guidance how to start volunteering. I encourage you to join our volunteers community Slack where you'll find many other people involved in building, translating, supporting and operating this community built platform.

---

(*) based on GitHub stats which include only code contributions, obviously there are other contributions too, which are harder to measure and grasp what does not make them less significant and valuable for the community. Being aware of their limits, GH stats acknowledge certain level of commitment and time investment.

**Comment by mariha** [comment]
I am especially curious to learn how did you @robokow come to [this conclution](https://github.com/Trustroots/trustroots/issues/2585#issuecomment-1269698630):
> The amusing thing is that TR foundation being dissolved doesn't mean anything to the community or for the community. TR has not changed; [...]

And if you have any thoughts about [a community being protected from the admins](https://github.com/Trustroots/trustroots/issues/2585#issuecomment-1269763412)?
> admins are here to safeguard TR from other volunteers taking decisions that can bring TR into jeopardy

**Comment by mariha** [comment]
Guys, let's make peace not war ✌️.

We meet usually on Mondays at 8pm CEST in  [trustroots room in jit.si](https://meet.jit.si/trustroots). Let's talk there.

**Comment by robokow** [comment]
> I'd like to add something to this convo onto why it is relevant to our members regarding trust and transparency.

Yes I agree with this argument you are raising. To me it's the wording that's the problem and the lack of consensus about those words, especially with the ones who actually have to push this commit to production.

> For the convo, I am especially curious to learn how did you come to https://github.com/Trustroots/trustroots/issues/2585#issuecomment-1269698630:

Those are for me the facts as they present themselves. There really is not a issue about this. The organisation can be revamped to life by the stroke of a single signature. It could have been done already, but there is an opportunity to let it rest for a bit until we've cleared our organisational structure, have a team that is capable of working together, and we can agree to a common goal. There is enough time, and there is no need to send out this message ¯\_(ツ)_/¯ to the community. In other words, we already know what it means.

Meanwhile I do think something could be written on the TR website about this, and I will help guiding that. But it takes a bit more time.

> And if you have any thoughts about a community being protected from the admins?

I think any project ultimately can fail because of a lack of volunteers, or because of people taking over a project, or because key people have to step aside and there is no clear hand-over, nor an explicit structure. 

In order to safeguard TR, I think we need a clear set of goals, clear communication structure, a common direction, a means for conflict-resolution and a structure that prevents anyone ever to push TR towards their own private goals. However, I don't know if it is actually possible to have an organisation that can rule out the latter. Maybe therefore a TR based on a P2P protocol where TR can be one of the many nodes, or is not even needed, is the common goal that can help us towards that vision of a self-governed community.

I will contact you shortly @mariha to see if we can have one to one call about this and anything else that's related.
